google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -217,25 +348,28 @@ def _get_blocks(blocks: List[documentai.Document.Page.Block], text: str) -> List
             from the document.
     Returns:
         List[Block]:
-             A list of Blocks.
+             A list of wrapped Blocks.
     """
     result = []
 
     for block in blocks:
         result.append(
             Block(
-                documentai_block=block,
+                documentai_object=block,
                 text=_text_from_layout(layout=block.layout, text=text),
+                _page=_page,
             )
         )
 
     return result
 
 
 def _get_paragraphs(
-    paragraphs: List[documentai.Document.Page.Paragraph], text: str
+    paragraphs: List[documentai.Document.Page.Paragraph],
+    text: str,
```

Comment:
For these helper functions too, we should not need to have a `text` argument any more.